charts/incubator/big-agi/values.yaml

```diff
@@ -0,0 +1,190 @@
+            OPENAI_API_ORG_ID: "{{ .Values.bigagi.llms.open_ai.api_org_id }}"
+            OPENAI_API_KEY: "{{ .Values.bigagi.llms.open_ai.api_key }}"
+            # Azure
+            AZURE_OPENAI_API_ENDPOINT: "{{ .Values.bigagi.llms.azure.api_host }}"
+            AZURE_OPENAI_API_KEY: "{{ .Values.bigagi.llms.azure.api_key }}"
+            # Anthropic
+            ANTHROPIC_API_HOST: "{{ .Values.bigagi.llms.anthropic.api_host }}"
+            ANTHROPIC_API_KEY: "{{ .Values.bigagi.llms.anthropic.api_key }}"
+            # Gemini
+            GEMINI_API_KEY: "{{ .Values.bigagi.llms.gemini.api_key }}"
+            # Mistral
+            MISTRAL_API_KEY: "{{ .Values.bigagi.llms.mistral.api_key }}"
+            # OpenRouter
+            OPENROUTER_API_KEY: "{{ .Values.bigagi.llms.openrouter.api_key }}"
+            # Perplexicity
+            PERPLEXITY_API_KEY: "{{ .Values.bigagi.llms.perplexity.api_key }}"
+            # TogetherAI
+            TOGETHERAI_API_KEY: "{{ .Values.bigagi.llms.together_ai.api_key }}"
+
+            # Model Observability: Helicone
+            HELICONE_API_KEY: "{{ .Values.bigagi.llms.helicone.api_host }}"
+
+            # Text-To-Speech
+            ELEVENLABS_API_KEY: "{{ .Values.bigagi.features.text_to_speech.elevenlabs.api_key }}"
+            ELEVENLABS_API_HOST: "{{ .Values.bigagi.features.text_to_speech.elevenlabs.api_host }}"
+            ELEVENLABS_VOICE_ID: "{{ .Values.bigagi.features.text_to_speech.elevenlabs.voice_id }}"
+            # Text-To-Image
+            PRODIA_API_KEY: "{{ .Values.bigagi.features.text_to_image.prodia.api_key }}"
+            # Google Custom Search
+            GOOGLE_CLOUD_API_KEY: "{{ .Values.bigagi.features.custom_search.google.api_key }}"
+            GOOGLE_CSE_ID: "{{ .Values.bigagi.features.custom_search.google.cse_id }}"
+            
+            # Backend Analytics
+            # BACKEND_ANALYTICS=
+
+            # Backend HTTP Basic Authentication (see `deploy-authentication.md` for turning on authentication)
+            # ingress + forwardAuth; lets not set these.
+            # HTTP_BASIC_AUTH_USERNAME=
+            # HTTP_BASIC_AUTH_PASSWORD=
+
+            # Frontend variables
+            # NEXT_PUBLIC_GA4_MEASUREMENT_ID=
+            # NEXT_PUBLIC_PLANTUML_SERVER_URL=
+  
+            # Database (MongoDB)
+            # not using it; just documenting it here.
+            # MDB_URI=
+
+  browser:
+    enabled: true
+    type: Deployment
+    strategy: RollingUpdate
+    replicas: 1
+    podSpec:
+      containers:
+        browser:
+          primary: true
+          enabled: true
+          imageSelector: browserImage
+          probes:
+            liveness:
+              enabled: true
+              type: http
+              path: /healthz
+              port: "{{ .Values.service.browser.ports.browser.port }}"
+            readiness:
+              enabled: true
+              type: http
+              path: /
```

Comment:
Why not healthz here and on start?

Reply:
i couldnt find the probes paths so set / as a fail safe